docs/upgrade/services_removal.md

```diff
@@ -0,0 +1,85 @@
+  ⮕ `ProductFacade::getAllProductSellingPricesIndexedByDomainId(Product $product)`
+- `ProductService::sortProductsByProductIds(array $products, array $orderedProductIds)`  
+  ⮕ `ProductRepository::getSortedProductsByIds($domainId, PricingGroup $pricingGroup, array $sortedProductIds)`
+- `ProductService::markProductForVisibilityRecalculation(Product $product)`  
+  ⮕ `Product::markForVisibilityRecalculation()`
+- `ProductService::edit(Product $product, ProductData $productData)`  
+  ⮕ `Product::edit(ProductCategoryDomainFactoryInterface $productCategoryDomainFactory, ProductData $productData, ProductPriceRecalculationScheduler $productPriceRecalculationScheduler)`
+- `ProductService::delete(Product $product)`  
+  ⮕ `Product::getProductDeleteResult()`
+- `ProductService::recalculateInputPriceForNewVatPercent(Product $product, $productManualInputPrices, $newVatPercent)`  
+  ⮕ `ProductManualInputPrice::recalculateInputPriceForNewVatPercent($inputPriceType, BasePriceCalculation $basePriceCalculation, InputPriceCalculation $inputPriceCalculation, $newVatPercent)`
+- `OrderService::getOrderDetailUrl(Order $order)`  
+  ⮕ `OrderUrlGenerator::getOrderDetailUrl(Order $order)`
+- `OrderService:: createOrderProductInOrder(Order $order, Product $product, Price $productPrice)`  
+  ⮕ `Order::createOrderProduct(Product $product, Price $productPrice, OrderProductFactoryInterface $orderProductFactory, Domain $domain, OrderPriceCalculation $orderPriceCalculation)`
+- `OrderService::editOrder(Order $order, OrderData $orderData)`  
+  ⮕ `Order::edit(OrderData $orderData, OrderItemPriceCalculation $orderItemPriceCalculation, OrderProductFactoryInterface $orderProductFactory, OrderPriceCalculation $orderPriceCalculation)`
+- `OrderStatusService::checkForDelete`  
+  ⮕ `OrderStatus::checkForDelete`
+
+Following classes have been removed:
+- `AdministratorService`
+- `AdministratorGridService`
+- `CustomerService`
+- `ProductService`
+- `OrderService`
+- `OrderStatusService`
+- `FlagService`
+
+Following methods have been removed:
+- `User::setDeliveryAddress()`, use `User::editDeliveryAddress()` instead
+- `Administrator::addGridLimit()`, use `Administrator::rememberGridLimit()` instead
+- `Administrator::removeGridLimit()`
+- `Administrator::getLimitByGridId()`
+- `FlagService::create()`, use `FlagFactory::create()` instead
+- `FlagService::edit()`, use `FlagFactory::edit()` instead
+- `ProductRepository::getOfferedByIds()`, use `ProductRepository::getSortedProductsByIds()` instead
+
+Following classes changed constructors:
+- `AdministratorFacade`
+- `AdministratorGridFacade`
+- `CachedBestsellingProductFacade`
+- `CustomerFacade`
+- `FlagFacade`
+- `OrderItemFacade`
+- `OrderProductFacade`
+- `OrderMailService`
+- `OrderEditResult`
+- `OrderFacade`
+- `ProductInputPriceFacade`
+- `ProductPriceRecalculator`
+- `ProductFacade`
+- `UserFactory`
+
+Follow also unclassified upgrade instructions:
```

Comment:
I believe we should also mention methods that changed their visibility. From this review, I remember at least one method that has changed the visibility from `public` to `protected` (but I do not remember which one was that :laughing:)